### PR TITLE
[improve] Bump project version to 3.0.0-SNAPSHOT

### PR DESCRIPTION
**Pull request by mattisonchao**
### Motivation

https://lists.apache.org/thread/pqkh9d784krtk0jrfcs164kwrl303m7l has approved. We can bump project version to 3.0.0-SNAPSHOT

### Modifications

- Bump project version to 3.0.0-SNAPSHOT

### Verifying this change

- [x] Make sure that the change passes the CI checks.

### Documentation

- [ ] `doc` 
- [ ] `doc-required` 
- [x] `doc-not-needed` 
- [ ] `doc-complete`

### CODE REVIEW

**Review by coderzc — APPROVED** [review]

File: buildtools/pom.xml [hunk 1]
```diff
@@ -31,12 +31,12 @@
...
   <groupId>org.apache.pulsar</groupId>
   <artifactId>buildtools</artifactId>
-  <version>2.12.0-SNAPSHOT</version>
+  <version>3.0.0-SNAPSHOT</version>
   <packaging>jar</packaging>
   <name>Pulsar Build Tools</name>
 
   <properties>
-    <project.build.outputTimestamp>${maven.build.timestamp}</project.build.outputTimestamp>
+    <project.build.outputTimestamp>2023-02-20T12:42:59Z</project.build.outputTimestamp>
```
Comment (gaoran10) [inline buildtools/pom.xml:39]: Why need to change this?

File: deployment/terraform-ansible/deploy-pulsar.yaml-e [hunk 2]
```diff
@@ -0,0 +1,215 @@
+#
```
Comment (gaoran10) [inline deployment/terraform-ansible/deploy-pulsar.yaml-e:1]: Why need to add this?

File: buildtools/pom.xml (hunk 1 above)
Comment (mattisonchao) [reply buildtools/pom.xml]: fixed.

File: deployment/terraform-ansible/deploy-pulsar.yaml-e (hunk 2 above)
Comment (mattisonchao) [reply deployment/terraform-ansible/deploy-pulsar.yaml-e]: fixed.

**Review by poorbarcode — APPROVED** [review]

**Review by gaoran10 — APPROVED** [review]

**Review by RobertIndie — APPROVED** [review]

**Pull request merged**